include/wlr/render/gles2.h

```diff
@@ -24,4 +25,16 @@ struct wlr_texture *wlr_gles2_texture_from_wl_drm(struct wlr_egl *egl,
 struct wlr_texture *wlr_gles2_texture_from_dmabuf(struct wlr_egl *egl,
 	struct wlr_dmabuf_attributes *attribs);
 
+struct wlr_gles2_texture_attribs {
+	GLenum target;
```

Comment:
Can we somehow let the library user know which targets to expect?

Reply:
Probably just a comment mentioning `GL_TEXTURE_2D` and `GL_TEXTURE_EXTERNAL_OES`. It'll never be anything other than that, even in the future, since all of the dmabuf type stuff is inherently 2D.

Reply:
That's a good point.